Assets/MixedRealityToolkit/Inspectors/ServiceInspectors/ServiceFacadeInspector.cs

```diff
@@ -362,15 +373,18 @@ private static void DrawSceneGUI(SceneView sceneView)
                     continue;
                 }
 
-                IMixedRealityServiceInspector inspectorInstance;
-                if (!GetServiceInspectorInstance(inspectorTypePair.Key, out inspectorInstance))
+                foreach (Type interfaceType in inspectorTypePair.Key.GetInterfaces())
                 {
```

Comment:
Similar comment here - how we loop over each interface type but we don't use it in this loop.

Reply:
Thank you, good catch.